### DISCUSSION TITLE
Strategy for merging PRs to main?

### DISCUSSION THREAD

**Discussion by grgp** [mgilangjanuar/teledrive#6] [Ideas]
Tbh I'm very opinionated on this haha. I really prefer using Squash & Merge.

![image](https://user-images.githubusercontent.com/12206156/131205202-b4d6d9ee-c011-4ebf-a357-7e6197f3a1c8.png)

"When you select the Squash and merge option on a pull request on GitHub, the pull request's commits are squashed into a single commit. Instead of seeing all of a contributor's individual commits from a topic branch, the commits are combined into one commit and merged into the default branch."

I like it because individual devs on PRs can have as many commits as they want, but when it's merged to master it's one cohesive commit with a clear purpose. Easy to revert, too.

**Comment by mgilangjanuar** [comment]
Yup, I agree with squash. **If** the current branch only has 1 contributor, to make us easy for blaming 🤣
